plugins/org.eclipse.mat.api/src/org/eclipse/mat/inspections/LeakHunterQuery.java

```diff
@@ -632,25 +632,26 @@ private String getRetainedHeapTopConsumersDescription(int[] objects, IProgressLi
         { return Messages.LeakHunterQuery_RetainedHeapComponentEmpty; }
         int[] minRetainedSet = snapshot.getMinRetainedSet(objects, listener);
 
-        Map<String, Long> sumHeapSizes = new HashMap<>();
-        Map<String, Integer> counts = new HashMap<>();
+        Map<Integer, Long> sumHeapSizes = new HashMap<>();
+        Map<Integer, Integer> counts = new HashMap<>();

```

Comment:
We also have `org.eclipse.mat.collect.HashMapIntLong` as an option, to avoid the boxing.